cmd/reviewdog/main.go

```diff
@@ -596,10 +597,19 @@ func gitlabClient(token string) (*gitlab.Client, error) {
 const defaultGitLabAPI = "https://gitlab.com/api/v4"
 
 func gitlabBaseURL() (*url.URL, error) {
-	baseURL := os.Getenv("GITLAB_API")
-	if baseURL == "" {
+	gitlabApi := os.Getenv("GITLAB_API")
+	gitlabV4Url := os.Getenv("CI_API_V4_URL")
```

Comment:
`gitlabV4URL`.

Also, please fix the above golint error as well.
> `var gitlabApi should be gitlabAPI`

Reply:
Done